main.cc

```diff
@@ -0,0 +1,210 @@
+#include <iostream>
+#include <string.h>
```

Comment:
The standard says that whenever we need to include a C header xxx.h in C++, we have to `#include <cxxx>`

Reply:
Great. I didn't know. Now I know :) Fixed.